flutter_maps_firestore/ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md

```diff
@@ -0,0 +1,5 @@
+# Launch Screen Assets
+
+You can customize the launch screen with your own desired assets by replacing the image files in this directory.
```

Comment:
Delete "own desired".

Reply:
Yup, we should update the templates in flutter/flutter for this =)